src/helpers/clipboard/navigator_clipboard_wrapper.ts

```diff
@@ -0,0 +1,65 @@
+import { ClipboardContent, ClipboardContentType } from "./../../types/clipboard";
+
+export type ClipboardReadResult =
+  | { status: "ok"; content: string }
+  | { status: "permissionDenied" | "notImplemented" };
+
+export interface ClipboardWrapperInterface {
+  writeContent(clipboardContent: ClipboardContent): Promise<void>;
+  writeText(text: string): Promise<void>;
+  readText(): Promise<ClipboardReadResult>;
+  clearClipboard(): Promise<void>;
+}
+
+export class ClipboardWrapper implements ClipboardWrapperInterface {
```

Comment:
Actually, I changed my mind. We don't care about the name since the world don't/shouldn't care about the implementation.
We could even go a step further to hide the implementation and have a factory function which would instantiate this class and return it, typed as the interface. The implementation wouldn't need to be exposed at all.

Reply:
Why not. Now `spreadsheet.ts` only call `instantiateClipboard()`, which instanciate a `WebClipboardWrapper` and returns a `ClipboardInterface`.